core/src/main/scala/scalaz/IList.scala

```diff
@@ -240,6 +240,9 @@ sealed abstract class IList[A] extends Product with Serializable {
   def map[B](f: A => B): IList[B] =
     foldRight(IList.empty[B])(f(_) :: _)
 
+  def foreach(f: A => Unit): Unit =
+    toList.foreach(f)
```

Comment:
Makes sense. However, I was just following what [NonEmptyList.foreach](https://github.com/scalaz/scalaz/blob/de094534fc3cda329b05c0995106c440c26f979e/core/src/main/scala/scalaz/NonEmptyList.scala#L25-L28) is doing.